eng/pipelines/agent-pool-migration.yml

```
@@ -0,0 +1,151 @@
+    - MigrateFrom: windows-2022
+      MigrateTo: windows-2022
+    - MigrateFrom: windows2022
+      MigrateTo: windows2022
+    - MigrateFrom: MMS2019
+      MigrateTo: MMS2022
+- name: ShortForMigrateFrom
+  type: string
+  default: ''
+- name: ShortForMigrateTo
+  type: string
+  default: ''
+# The value here is github alias: e.g. sima-zhu
+- name: GithubAssigneeAlias
+  type: string
+  default: ''
+- name: PRForkOwner
+  type: string
+  default: 'azure-sdk'
+- name: WhatIfPreference
+  type: boolean
+  default: false
+
+variables:
+  - template: /eng/pipelines/templates/variables/globals.yml
+jobs:
+  - job: Migration
+    pool:
+      name: azsdk-pool-mms-win-2022-general
+    steps:
+      - ${{ each repo in parameters.Repos }}:
+        - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+          parameters:
+            SkipCheckoutNone: true
+            Repositories:
+              - Name: ${{ repo.RepoOwner }}/${{ repo.RepoName }}
+                WorkingDirectory: $(Build.SourcesDirectory)/repos/${{ repo.RepoName }}
+                Commitish: ${{ repo.branch }}
+            Paths:
+              - /*
+        - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
+          parameters:
+            WorkingDirectory: $(Build.SourcesDirectory)/repos/${{ repo.RepoName }}
+        - pwsh: |
+            $migrationMap = '${{ convertToJson(parameters.MigrationMap) }}'
+            $(Build.SourcesDirectory)/eng/scripts/Repo-File-Content-Replacements.ps1 `
+              -ExcludePathsRegex '${{ repo.ExcludePaths }}' `
+              -IncludePathsRegex '${{ repo.IncludePaths }}' `
+              -MigrationMap $migrationMap
+          displayName: Migration for "${{ repo.RepoName }}"
+          workingDirectory: $(Build.SourcesDirectory)/repos/${{ repo.RepoName }}
+          continueOnError: true
+        - ${{ if ne(parameters.WhatIfPreference, 'true')}}:
+          - pwsh: |
+              $requestBy = '${{ parameters.GithubAssigneeAlias }}'
+              if ($requestBy) {
+                Write-Host "##vso[task.setvariable variable=UserReviewers]$requestBy"
+              }
+            displayName: Set Github reviewers variable
+          - template: /eng/common/pipelines/templates/steps/create-pull-request.yml
+            parameters:
+              BaseBranchName: $(DefaultBranch)
+              PRBranchName: migration
+              PROwner: ${{ parameters.PRForkOwner }}
+              CommitMsg: "Migrate from ${{parameters.ShortForMigrateFrom}} to ${{parameters.ShortForMigrateTo}}"
+              RepoOwner: ${{ repo.RepoOwner }}
+              RepoName: ${{ repo.RepoName }}
+              PRTitle: "Migrate from ${{parameters.ShortForMigrateFrom}} to ${{parameters.ShortForMigrateTo}}"
+              WorkingDirectory: $(Build.SourcesDirectory)/repos/${{ repo.RepoName }}
+              ${{ if ne(parameters.GithubAssigneeAlias, '')}}:
+                GHReviewersVariable: UserReviewers
```

```suggestion
                GHAssignessVariable: UserReviewers
```

Shouldn't it be Assignees? Looks like we also have a typo in that parameter name https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/pipelines/templates/steps/create-pull-request.yml#L18